### PR TITLE
Update documentation to use new `tanzu-cli-installer-packages` repo for APT/RPM packages and new Public Key

### PR DESCRIPTION
**Pull request by anujc25**
### What this PR does / why we need it

- Update docs to use new `tanzu-cli-installer-packages` repo for APT/RPM packages
- Update docs to use new Public Key

### Which issue(s) this PR fixes

Fixes #

### Release note

```release-note
Update docs to use new `tanzu-cli-installer-packages` repo for APT/RPM packages and new Public Key
```

### CODE REVIEW

**Review by marckhouzam — APPROVED** [review]
This looks good.
Should we make a similar change under hack/rpm/README.md and hack/apt/README.md?

**Comment by vuil** [comment]
> This looks good. Should we make a similar change under hack/rpm/README.md and hack/apt/README.md?

Good catch, we should.

File: cmd/plugin/builder/template/plugintemplates/github_workflow_build.yaml.tmpl [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
       run: |
         sudo apt-get update
         sudo apt-get install -y ca-certificates
-        sudo echo "deb https://storage.googleapis.com/tanzu-cli-os-packages/apt tanzu-cli-jessie main" | sudo tee /etc/apt/sources.list.d/tanzu.list
+        sudo echo "deb https://storage.googleapis.com/tanzu-cli-installer-packages/apt tanzu-cli-jessie main" | sudo tee /etc/apt/sources.list.d/tanzu.list
```
Comment (marckhouzam) [inline cmd/plugin/builder/template/plugintemplates/github_workflow_build.yaml.tmpl:29]: Good catch!

**Review by marckhouzam — APPROVED** [review]
Nice!

**Pull request merged**